upgrading/topics/keycloak/changes.adoc

```diff
@@ -1,5 +1,19 @@
 == Migration Changes
 
+=== Migrating to 8.0.1
```

Comment:
probably it should be updated to '8.0.2' since 8.0.1 doesn't include this change